dotnet/samples/ApplicationInsightsExample/Program.cs

```diff
@@ -33,75 +30,61 @@ public sealed class Program
     /// </remarks>
     private const LogLevel MinLogLevel = LogLevel.Information;
 
+    /// <summary>
+    /// Instance of <see cref="ActivitySource"/> for the application activities.
+    /// </summary>
+    private static readonly ActivitySource s_activitySource = new("ApplicationInsights.Example");
+
     /// <summary>
     /// The main entry point for the application.
     /// </summary>
     /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
     public static async Task Main()
     {
-        var serviceProvider = GetServiceProvider();
+        var connectionString = Env.Var("ApplicationInsights__ConnectionString");
 
-        var telemetryClient = serviceProvider.GetRequiredService<TelemetryClient>();
-        var loggerFactory = serviceProvider.GetRequiredService<ILoggerFactory>();
+        using var traceProvider = Sdk.CreateTracerProviderBuilder()
+            .AddSource("Microsoft.SemanticKernel.*")
+            .AddSource("ApplicationInsights.Example")
+            .AddAzureMonitorTraceExporter(options => options.ConnectionString = connectionString)
+            .Build();
 
-        using var meterListener = new MeterListener();
-        using var activityListener = new ActivityListener();
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
```

Comment:
Are the traceProvider and meterProvider variables used at all or they not need to be used?

Reply:
They don't need to be used explicitly but the instance needs to be kept active throughout the process lifetime: https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-configuration?tabs=net